readme.txt

```diff
@@ -1,18 +1,16 @@
 === WP LCache ===
 Contributors: getpantheon, danielbachhuber, stevector
-Tags: cache, plugin
+Tags: apc, apcu, cache, caching, lcache, object-cache, performance
 Requires at least: 4.3
-Tested up to: 4.7.4

```

Comment:
> Because WordPress ignores minor releases in this field so you aren't required to bump your versions at every point release, making it just added verboseness.

Makes sense.